soc/xtensa/intel_adsp/common/boot.c

```diff
@@ -41,10 +41,16 @@
 #define HOST_PAGE_SIZE		4096
 #define MANIFEST_SEGMENT_COUNT	3
 
+/* FIXME: Use Kconfig or some other means */
 #if defined(CONFIG_SOC_INTEL_CAVS_V15)
 #define PLATFORM_DISABLE_L2CACHE_AT_BOOT
 #endif
 
+/* FIXME: Use Kconfig or some other means */
+#if !defined(CONFIG_SOC_SERIES_INTEL_ACE1X)
+#define RESET_MEMORY_HOLE
```

Comment:
FWIW: this "memory hole" business was copied from SOF more or less blindly.  I forget all the details, but it never really made sense vs. the description of that L2MECS register.  This might be ejectable, or there might some hardware errata that's been forgotten, etc...